src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -281,6 +286,17 @@ else if (callVariantContextType == INDEL) {
             progress.record(variantContextForLogging.getContig(), variantContextForLogging.getStart());
         }
 
+        //snp counter add in X number of missing-missing hom ref's (truth and call state)
+        //missing missing is total interval size minus number of iterations in while loop
+
+        if(MISSING_SITES_HOM_REF && usingIntervals){
+            addMissingMissingSites(snpCounter.getCounterSize(), intervalBaseCount, snpCounter);
+            addMissingMissingSites(indelCounter.getCounterSize(), intervalBaseCount, indelCounter);
+        }
+        else if(MISSING_SITES_HOM_REF && !usingIntervals) {
+            throw new PicardException("You cannot use the MISSING_SITES_HOM_REF flag without using a high confidence interval list");
```

Comment:
This check should be moved further up in this method (to where the other fail fast checks are).  It would be a shame to do tons of work only to find out at the end that the user neglected to provide high confidence intervals.

Then, given that if MISSING_SITES_HOM_REF is true here we know that usingIntervals must be true (otherwise you would have errored out earlier in the method), the if statement a few lines earlier (line 292) now only needs to be:
if ( MISSING_SITES_HOM_REF )

Finally, I would change the error message to something like:
"You cannot use the MISSING_HOM option without also supplying an interval list over which missing sites are considered confident homozygous reference calls."